### DISCUSSION TITLE
Problems getting file download file authorised: Status 403

### DISCUSSION THREAD

**Discussion by drummonds** [photoprism/photoprism#4496] [Q&A]
Dear Photoprism
I am trying to get a go library to work which I generated from your swagger io.  Trying to run an example app which dowloads the first 10 pictures of the first album works except for https://github.com/drummonds/photoprism-go-api/blob/main/example/showAlbum/showAlbum.go#L88
The download of the file.

In order to get it to work I need to set the PHOTOPRISM_AUTH_MODE to public (which is actually fine in my case) which is not ideal.
I am using an X-AUTH_TOKEN which is working fine for getting data but not dowloading the files.

**Comment by graciousgrey** [comment]
Thank you for your feedback! Please note that the swagger docs are still work in progress, so not all endpoints are documented in detail yet. Which endpoint are you trying to use?
